scripts/prepare_test_env.sh

```diff
@@ -0,0 +1,16 @@
+export DOMAIN_NAME=$1
+export DEX_DOMAIN_NAME="dex.$DOMAIN_NAME"
+export MGMT_DOMAIN_NAME="mgt.$DOMAIN_NAME"
+echo "Fetching CA for $MGMT_DOMAIN_NAME"
+rm -fr ca.pem
```

Comment:
since you're using `> ca.pem`  we dont have to remove this file

Reply:
👍